### PR TITLE
Format logs as JSON

### PR DESCRIPTION
**Pull request by Bernardstanislas**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [package]
 authors = ["Abdelhamid Bakhta <abdelhamid.bakhta@gmail.com>"]
 categories = ["contribution", "health-check", "analytics-tracking", "analytics"]
+default-run = "deathnote_contributions_feeder"
```
Comment (AnthonyBuisset) [inline Cargo.toml:4]: Shouldn't we rename to marketplace?

File: src/main.rs [hunk 2]
```diff
@@ -36,7 +37,14 @@ embed_migrations!("migrations");
 #[tokio::main]
 async fn main() {
 	dotenv().ok();
-	env_logger::init();
+	let drain = slog_async::Async::default(slog_envlogger::new(
```
Comment (AnthonyBuisset) [inline src/main.rs:40]: Could we refactor this into a function to explain what it does ?

**Review by AnthonyBuisset — APPROVED** [review]
LGTM.
Left some minor comments

File: Cargo.toml (hunk 1 above)
Comment (Bernardstanislas) [reply Cargo.toml:4]: Yes I tried to do it in this PR but it's too big to mix both subjects

File: Cargo.toml [hunk 3]
```diff
@@ -39,29 +39,35 @@ octocrab = "0.16.0"
...
+okapi = {version = "0.7.0-rc.1"}
+rocket = {version = "0.5.0-rc.2", features = ["json"]}
+rocket_okapi = {version = "0.8.0-rc.2", features = ["swagger"]}
 schemars = "0.8"
 
 # Starknet
-starknet = { git = "https://github.com/ofux/starknet-rs" }
-crypto-bigint = { version = "0.4.8", features = ["serde"] }
+crypto-bigint = {version = "0.4.8", features = ["serde"]}
+starknet = {git = "https://github.com/ofux/starknet-rs"}
```
Comment (tdelabro) [inline Cargo.toml:50]: I think olivier PR was merged into the main repo. We can go back to it

Comment (Bernardstanislas) [reply Cargo.toml:50]: They've merged it but I'm not sure they've created a new release for it https://github.com/xJonathanLEI/starknet-rs/tags

File: src/main.rs (hunk 2 above)
Comment (Bernardstanislas) [reply src/main.rs:38]: updated

File: Cargo.toml (hunk 3 above)
Comment (ofux) [reply Cargo.toml:50]: They have indeed merged it, and we point to "https://github.com/ofux/starknet-rs" which means we take the last commit, not the last release :) So yeah, we can switch back to the main repo.
Sooner or later, we'll have to use releases.

**Pull request merged**